frontend/packages/data-portal/app/routes/browse-data.runs.tsx

```diff
@@ -0,0 +1,5 @@
+import { Demo } from 'app/components/Demo'
+
+export default function BrowseRunsPage() {
+  return <Demo>Browse runs page</Demo>
+}
```

Comment:
since this is a demo, maybe add a TODO to replace it with the real thing later? (idk just nitpicking)